packages/node/src/integrations/requestdata.ts

```diff
@@ -18,6 +15,7 @@ export type RequestDataIntegrationOptions = {
     ip?: boolean;
     query_string?: boolean;
     url?: boolean;
+    // TODO(v8): Think really hard about whether we want this integration to concern itself with user data. Likely not, so we should remove it.
```

Comment:
I think I would avoid these comments, but rather discuss this in an issue? otherwise if we decide to not do this in v8 this comment will hang around weirdly xD